src/frontend/js/components/CourseGlimpse/utils.ts

```diff
@@ -14,17 +14,16 @@ const getCourseGlimpsePropsFromCourseProductRelation = (
   intl: IntlShape,
   organizationId?: string,
 ): CourseGlimpseCourse => {
-  const getRoutePath = getDashboardRoutePath(intl);
   const courseRouteParams = {
     courseId: courseProductRelation.course.id,
     courseProductRelationId: courseProductRelation.id,
   };
   const courseRoute = organizationId
-    ? getRoutePath(TeacherDashboardPaths.ORGANIZATION_PRODUCT, {
+    ? getDashboardRoutePath(TeacherDashboardPaths.ORGANIZATION_PRODUCT, {
```

Comment:
Why not using `generatePath` directly ?

Reply:
I preferred to keep a bit of encapsulation, what do you think @jbpenrath? I'll be democratic on this

Reply:
As now, this method acts as a proxy, I'll use `generatePath` directly.